### PR TITLE
code giao dien va tinh nang tab tracks

### PR DESCRIPTION
**Pull request by trantan97**
code giao dien va tinh nang tab tracks
https://edu-redmine.framgia.vn/issues/5718

### CODE REVIEW

**Comment by trantan97** [comment]
Ready

File: app/src/main/java/com/trantan/music53/ui/tracks/TracksFragment.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    private ServiceConnection mConnection;
+    private List<Track> mTracks;
+
+    public TracksFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onStart() {
+//        Log.d(TAG, "onStart: ");
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/trantan/music53/ui/tracks/TracksFragment.java]: remove comment

File: app/src/main/java/com/trantan/music53/ui/tracks/TracksFragment.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        super.onStart();
+    }
+
+    @Override
+    public void onPause() {
+//        Log.d(TAG, "onPause: ");
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/trantan/music53/ui/tracks/TracksFragment.java]: remove comment

**Comment by trantan97** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**